doc/source/scipyoptdoc.py

```diff
@@ -82,12 +83,17 @@ def run(self):
             # Interface function
             name = self.arguments[0].strip()
             obj = _import_object(name)
-            args, varargs, keywords, defaults = inspect.getargspec(obj)
+            # XXX deprecation that we should fix eventually
+            with warnings.catch_warnings(record=True):
+                warnings.simplefilter('ignore')
+                args, varargs, keywords, defaults = inspect.getargspec(obj)
```

Comment:
Is the deprecarion in inspect.getargspec? There's a shim in https://github.com/scipy/scipy/blob/master/scipy/_lib/_util.py#L249

Reply:
Two of the three are, yeah, thanks. I've replaced those.